src/System.Text.Json/experiments/Json.Net.Test/Json.Net.Tests/JsonParser.cs

```diff
@@ -0,0 +1,565 @@
+﻿using System;
+
+namespace Json.Net.Tests
+{
+    public struct JsonParser : IDisposable
```

Comment:
How are we going to keep the sources in sync? I would try to find a way to remove this file.